.github/workflows/build-docs.yml

```diff
@@ -88,29 +88,48 @@ jobs:
         continue-on-error: true
         env:
           VERSION: ${{ github.event.release.tag_name }}
+          JVM_PROJECTS: |
+            jans-auth-server
+            jans-orm
+            jans-config-api
+            jans-scim
+            jans-core
+            jans-notify
+            jans-fido2
+            jans-eleven
+            agama
         run: |
          echo "Custom work on generating docs can go here."
           
          # Run cn docs 
          sudo bash ./automation/docs/generated-cn-docs.sh
```

Comment:
Just noticed that `generate-cn-docs.sh` doesn't add or commit anything to git. Is that how it should be?

Reply:
yes the git operations shouldn't be in the bash scripts unless needed.